### PR TITLE
Fix csg link in inversion_directe_salaires

### PR DESCRIPTION
**Pull request by benjello**

### CODE REVIEW

**Comment by benjello** [comment]
You do not use this reform but you may have comments.

**Comment by benjello** [comment]
@fpagnoux @cbenz : I would like to merge this ASAP

File: openfisca_france/reforms/inversion_directe_salaires.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 # import logging
 
-from openfisca_core import columns, formulas, reforms
+from openfisca_core import columns, reforms
```
Comment (fpagnoux) [inline openfisca_france/reforms/inversion_directe_salaires.py]: Why not
```py
 from openfisca_france.model.base import *  # noqa analysis:ignore
```
?

Comment (benjello) [inline openfisca_france/reforms/inversion_directe_salaires.py]: history ;-)

Comment (benjello) [inline openfisca_france/reforms/inversion_directe_salaires.py]: ongoing

**Comment by benjello** [comment]
@fpagnoux is this GTM ?

**Comment by benjello** [comment]
I am merging this PR

**Comment by fpagnoux** [comment]
👌

**Pull request merged**